### ISSUE TITLE
Clearing a move count result is intercepted

### ISSUE THREAD

**Issue by lgarron** [cubing/cubingcontests.com#60]
1. Select the text of a move count result like "50".
2. Press backspace to clear the result (e.g. to type in a new one).

Expected: End up with a clear field.
Observed: Only the last character is deleted.

Probably the same root cause as #58.

**Comment by mintydev789** [comment]
Yeah, this is the intended behavior, as the input is completely restricted and only allows editing it one digit at a time. If you believe this behavior should be changed to make it work as a standard text input specifically for the `number` event format (as opposed to `time` or `multi`), please open an issue on [Codeberg](https://codeberg.org/mintydev/RecordRanks).
